_posts/2019-04-12-rails-girls-buenos-aires-2019.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title:  "Rails Girls: growing as a community"
+date: 2019-04-12 10:00:00
+categories: ["learning", "rails"]
+author: "luciano"
+---
+
+In a [previous article](https://www.ombulabs.com/blog/learning/rails/another-successful-rails-girls-event.html) we talked about one of the [Rails Girls](http://railsgirls.com) events that we did in 2018. Fast forward one year to our first event of 2019 at [A3](https://areatresworkplace.com), a new coworking space located in Buenos Aires.
+
+<!--more-->
+
+Our goal this year was to go further and try to reach around 50 participants (almost twice than previous years). Gratefully we had a good number of volunteer coaches that were willing to help us make that possible.
+
+Some of the coaches were girls that participated in a previous event and now they took the role of coach so they can share their knowledge that they've been acquiring during the last couple of months. We find that really motivating during these events.
```

Comment:
```
were girls that
```

maybe this sounds better:

```
were women who
```

Reply:
This is a monster (just because it is too long) sentence:

```
Some of the coaches were girls that participated in a previous event and now they took the role of coach so they can share their knowledge that they've been acquiring during the last couple of months. 
```

Could you try breaking it into two sentences?